actors/builtin/reward/reward_actor.go

```diff
@@ -142,15 +139,20 @@ func (a Actor) UpdateNetworkKPI(rt vmr.Runtime, currRealizedPower *abi.StoragePo
 
 	var st State
 	rt.State().Transaction(&st, func() interface{} {
-		newBaselinePower := a.newBaselinePower(&st)
-		st.BaselinePower = newBaselinePower
-		st.CumsumBaseline = big.Add(st.CumsumBaseline, st.BaselinePower)
+		for i := st.LastKPIUpdate; i < rt.CurrEpoch(); i++ {
+			newBaselinePower := a.newBaselinePower(&st)
+			st.BaselinePower = newBaselinePower
+			st.CumsumBaseline = big.Add(st.CumsumBaseline, st.BaselinePower)
+
+			st.RealizedPower = *currRealizedPower
+			st.CumsumRealized = big.Add(st.CumsumRealized, *currRealizedPower)
```

Comment:
We dont want to add `currRealizedPower` to `CumsumRealized` unless `i == rt.CurrEpoch` or `i == rt.CurrEpoch - 1`. In other words, for a null round, we are only adding the previous `st.RealizedPower` to the cumsum as no new power was proven.

Reply:
I think @whyrusleeping misinterpreted this. We should add _something_ (`st.RealizedPower`), just not the new `currRealizedPower`